sdks/go/pkg/beam/runners/dataflow/dataflowlib/translate.go

```diff
@@ -117,7 +118,7 @@ func (x *translator) translateTransform(trunk string, id string) ([]*df.Step, er
 		// URL Query-escaped windowed _unnested_ value. It is read back in
 		// a nested context at runtime.
 		var buf bytes.Buffer
-		if err := exec.EncodeWindowedValueHeader(exec.MakeWindowEncoder(coder.NewGlobalWindow()), window.SingleGlobalWindow, mtime.ZeroTimestamp, &buf); err != nil {
+		if err := exec.EncodeWindowedValueHeader(exec.MakeWindowEncoder(coder.NewGlobalWindow()), window.SingleGlobalWindow, mtime.ZeroTimestamp, typex.PaneInfo{}, &buf); err != nil {
```

Comment:
Question: Is the zero value for your pane info type identical to the "No firing" pane we've been writing? If so, very nice!

Optionally, also add a function to the typex package: typex.NoFiringPane() to document the intended semantic meaning of the type at the call sights like this.